appveyor.yml

```diff
@@ -58,7 +56,7 @@ for:
       - ps: .\install_windows.ps1
     test_script:
     - ps: |
-        git checkout master
+        git checkout ${DRIVER_LATEST_TAG}
```

Comment:
I think this doesn't work on powershell, should be $env:DRIVER_LATEST_TAG

Reply:
Actually I have to fix the target frameworks part since the last changes on the C# driver tests, I'll open a PR for that and I'll fix this issue too

Reply:
oh good catch, sorry I messed up.